src/ui/units/wizard/components/Dialogs/DialogColor/DialogColor.tsx

```diff
@@ -54,21 +55,36 @@ export type OpenDialogColorArgs = {
     props: OwnProps;
 };
 
-class DialogColorComponent extends React.Component<Props> {
+interface State {
+    colorMode: ColorMode;
+}
+
+class DialogColorComponent extends React.Component<Props, State> {
+    constructor(props: Props) {
+        super(props);
+
+        // isGradient is legacy fallback flag
+        // colorMode is modern value
+        this.state = {
+            colorMode:
+                props.colorsConfig?.colorMode ||
+                (props.isGradient ? ColorMode.GRADIENT : ColorMode.PALETTE),
```

Comment:
Fields that could have been preserved in the old features need to be mapped through the config migration mechanism.

Reply:
If we want to do it that way, we should add v10 version, because now acutal version is v9.